bootstrap/index.ts

```diff
@@ -31,16 +31,24 @@
         const aliucordDir = `${externalStorageDirectory}/AliucordRN`;
         AliuFS.mkdir(aliucordDir);
 
-        const commits = await fetch("https://api.github.com/repos/Aliucord/AliucordRN/commits?sha=builds");
-        const json = await commits.json();
-        const latestCommit = json[0].sha;
-
         const externalBundlePath = `${aliucordDir}/Aliucord.js.bundle`;
-        const internalBundlePath = `${codeCacheDirectory}/${latestCommit}.js.bundle`;
         if (AliuFS.exists(externalBundlePath)) {
             globalThis.aliucord = AliuHermes.run(externalBundlePath);
             return;
         }
+
+        const commits = await fetch("https://api.github.com/repos/Aliucord/AliucordRN/commits?sha=builds&path=bootstrap.js&per_page=1");
+        if (commits.status === 403) {
+            const internalBundlePath = `${codeCacheDirectory}/Aliucord.js.bundle`;
+            if (!AliuFS.exists(internalBundlePath)) await download("https://raw.githubusercontent.com/Aliucord/AliucordRN/builds/Aliucord.js.bundle", internalBundlePath);
```

Comment:
you'd need to download every time here cause therea no way to tell if it's old

Reply:
yop, was going to ask.